doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -508,8 +520,8 @@ The validation can then be based on simple pass/fail test results or on a grade
 
 Two categories of metrics are considered depending on whether they are related to either the VNF domain or the NFVI domain itself:
 
-- Metrics related to the VNF domain are defined from VNF perspective (i.e., per VNFC, per vNIC, per vCPU...) and should concern VNF as well as NFVI actors. 
-- Metrics related to the NFVI domain are defined per NFVI node ; their measurement is based on virtual workloads (i.e., VM or container) in order to reflect the performance of a NFVI node with a given profile ; they should only concern NFVI actors. 
+- Metrics related to the VNF domain are defined from VNF perspective (i.e., per VNFC, per vNIC, per vCPU...) and should concern VNF as well as NFVI actors.
+- Metrics related to the NFVI domain are defined per NFVI node ; their measurement is based on virtual workloads (i.e., VM or container) in order to reflect the performance of a NFVI node with a given profile ; they should only concern NFVI actors.
```

Comment:
can you clarify, ".. NFVI node with a given profile"

typo extra space before ";"